guides/source/active_record_migrations.md

```diff
@@ -1128,10 +1388,9 @@ state of your database schema.
 
 It tends to be faster and less error prone to create a new instance of your
 application's database by loading the schema file via `bin/rails db:schema:load`
-than it is to replay the entire migration history.
-[Old migrations][] may fail to apply correctly if those migrations use changing
-external dependencies or rely on application code which evolves separately from
-your migrations.
+than it is to replay the entire migration history. [Old migrations][] may fail
+to apply correctly if those migrations use changing external dependencies or
+rely on application code which evolves separately from your migrations.
 
 Schema files are also useful if you want a quick look at what attributes an
```

Comment:
I use schema.rb constantly for this purpose. Might be worth calling this out in a TIP or something?